integration/operate-first/README.md

```diff
@@ -0,0 +1,92 @@
+
+### Logging In 
+You can access the Smaug cluster with this [OpenShift console login link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/authorize?client_id=console&redirect_uri=https%3A%2F%2Fconsole-openshift-console.apps.smaug.na.operate-first.cloud%2Fauth%2Fcallback&response_type=code&scope=user%3Afull&state=98ae2ceb). Click on `operate-first` to login with GitHub authentication. Once logged into the OpenShift console, you can use [this link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/token/display) to get an `oc login` command with a token that will let you login to the Smaug OpenShift cluster from your terminal. 
+
+## Deploying Fybrik Cluster-Scoped Resources
+This `integration/operate-first` directory contains the raw YAML files of the cluster scoped resources deployed for Fybrik. These files are generated from the Helm charts in `charts/fybrik`.
+
+If the Helm chart has been updated, follow the steps below to generate the new yaml files:
+1. Install yq and Helm in this repo. Run these commands from the root directory of this repo:
+```bash
+cd hack/tools
+./install_yq.sh
+./install_helm.sh
+```
+2. Go back to the `integration/operate-first` folder and set up the Python environment there
+```bash
+cd samples/operate-first
+pipenv install
+pipenv shell
+```
+3. Run Makefile to generate new YAML files from the Helm charts:
+```bash
+make all
+```
+
+After the cluster-scoped YAML files are generated, create a PR to the [operate-first/apps](https://github.com/operate-first/apps) repository in the [cluster-scope/base](https://github.com/operate-first/apps/tree/master/cluster-scope/base) directory with the YAML files in the subdirectories organized by resource type. More documentation about contributing to the `operate-first/apps` repository can be found [here](https://github.com/operate-first/apps/tree/master/docs/content)
+
+## Deploying Namespace-Scoped resources
+Operate First has an [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) instance deployed on MOC that can be used to deploy OpenShift resources located on a Git Repository. Fybrik has been onboarded to ArgoCD by following [these instructions](https://github.com/operate-first/apps/blob/master/docs/content/argocd-gitops/onboarding_to_argocd.md) and an ArgoCD project has been created for Fybrik. You can login to [the ArgoCD instance](https://argocd.operate-first.cloud/applications?proj=&sync=&health=&namespace=&cluster=&labels=) with the same login method as above. We have deployed 3 [ArgoCD applications](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#applications) which are automatically synced with the latest release of Fybrik. See the ArgoCD applications below pointing to the corresponding Fybrik repositories:
+* fybrik -> https://github.com/fybrik/charts
```

Comment:
looking at the application yaml it does not include fybrik-crd, can you please explain why fybrik-crd is deployed differently? Thanks

Reply:
Added